src/controllers/applications.ts

```diff
@@ -0,0 +1,38 @@
+import { ApplicationService } from "../service/application-service";
+import { ApplicationParams } from "../util/types";
+
+export class Applications {
+  async createApplication(applicationParams: ApplicationParams) {
+    try {
+      return await new ApplicationService().createApplication(
+        applicationParams
+      );
+    } catch (err) {
+      console.error(err);
```

Comment:
Try not to use silent errors, throw an appropriate error. This needs action everywhere across the project.